src/api/errors/levels/BaseError.ts

```diff
@@ -0,0 +1,27 @@
+class BaseError extends Error {
+  type: string
+  code: number
+  title: string
+  level: string
+
+  constructor (code: number, title: string, message: string, level: string){
```

Comment:
Mind spaces between brackets
```suggestion
  constructor(code: number, title: string, message: string, level: string) {
```